build.gradle.kts

```diff
@@ -1,27 +1,27 @@
 import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
-val dusseldorfKtorVersion = "1.5.1.609bb61"
+val dusseldorfKtorVersion = "1.5.2.5b2dff7"
 val k9FormatVersion = "5.1.16"
 val ktorVersion = ext.get("ktorVersion").toString()
 val slf4jVersion = ext.get("slf4jVersion").toString()
 val kotlinxCoroutinesVersion = ext.get("kotlinxCoroutinesVersion").toString()
 
-val openhtmltopdfVersion = "1.0.6"
+val openhtmltopdfVersion = "1.0.8"
 val kafkaEmbeddedEnvVersion = "2.4.0"
 val kafkaVersion = "2.4.0" // Alligned med version fra kafka-embedded-env
```

Comment:
Henter du også siste versjon fra Düsseldorf-ktor?

Reply:
Aha ja! Bruke "maven("https://jitpack.io")" for å takle kafkaEmbeddedEnvVersion  2.7.0?

Reply:
Det stemmer. 
Du kan samtidig fjerne jcenter(), da den utgår.
https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/